tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="namelist_defaults.xsl"?>
+
+<rawdata_files>
+
+  <!-- ======================== -->
+  <!-- lai -->
+  <!-- ======================== -->
+
+  <mksrf_flai>
+    <entry>
+      <data_filename>lnd/clm2/rawdata/pftcftlandusedynharv.0.25x0.25.MODIS.simyr1850-2015.c170412/mksrf_lai_78pfts_simyr2005.cdf5.c220206.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_0.25x0.25_nomask_c200129.nc</mesh_filename>
+    </entry>
+    <entry pft_years="2005" hires_pft='on'>
+      <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.05x0.05.LUH2.histsimyr2005.c190116/mksrf_lai_histclm52deg005_earthstatmirca_2005.cdf5.c220228.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_3x3min_nomask_c200129.nc</mesh_filename>
+    </entry>
+  </mksrf_flai>
+
+  <!-- ======================== -->
+  <!-- soil properties: forganic, fmax, soil-texture and soil color-->
+  <!-- ======================== -->
+
+  <mksrf_forganic>
+    <entry>
+      <data_filename>lnd/clm2/rawdata/mksrf_organic_10level_5x5min_ISRIC-WISE-NCSCD_nlev7_c120830.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_5x5min_nomask_c200129.nc</mesh_filename>
+    </entry>
+  </mksrf_forganic>
+
+  <!-- <mksrf_firrig> -->
+  <!--   <entry> -->
+  <!--     <data_filename>lnd/clm2/rawdata/mksrf_irrig_2160x4320_simyr2000.c110527.nc</data_filename> -->
+  <!--     <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_5x5min_nomask_c200129.nc</mesh_filename> -->
+  <!--   </entry> -->
+  <!-- </mksrf_firrig> -->
+
+  <mksrf_fsoicol>
+    <entry>
+      <data_filename>lnd/clm2/rawdata/pftcftlandusedynharv.0.25x0.25.MODIS.simyr1850-2015.c170412/mksrf_soilcolor_CMIP6_simyr2005_cd5.c220126.nc</data_filename>
```

Comment:
@lawrencepj1 just to confirm, should this be updated to
`pftcftdynharv.0.05x0.05.LUH2.histsimyr2005.c190116/mksrf_soilcolor_histclm52deg005_earthstatmirca_2005.cdf5.c220228.nc`?

Reply:
Thanks @ekluzek and @slevisconsulting yes this is the correct file for updated soil color values

Reply:
...actually, let me clarify further:
We're already pointing to the new one for `<entry pft_years="2005" hires_pft='on'>` and otherwise we're pointing to the file shown in line 42. Should we now always point to the new file?

Reply:
Actually that isn't the correct file, that's the file only for use for the hirespft option. @lawrencepj1 is going to update that one with new LAI and soil color datasets that are all at the 0.05x0.05 resolution. For the standard resolution case there is a 0.25x0.25 resolution dataset that goes with the PFT files.

Reply:
@slevisconsulting this is still an issue, but I'm creating an issue for it to make sure we resolve it. Since, we are getting many new datasets we just need to make sure the final datasets are the right ones.

Reply:
@slevisconsulting scratch that -- it's correct.